vgc/io/outputAdapter.py

```diff
@@ -1,8 +1,19 @@
+import base64
```

Comment:
Should list all classes, exceptions, functions and objects that belong to the module.